buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -290,20 +290,20 @@ class ClusterFormationTasks {
         Copy copyConfig = project.tasks.create(name: name, type: Copy, dependsOn: setup)
         File configDir = new File(node.homeDir, 'config')
         copyConfig.into(configDir) // copy must always have a general dest dir, even though we don't use it
-        for (Map.Entry<String,Object> extraConfigFile : node.config.extraConfigFiles.entrySet()) {
```

Comment:
It isn't cosmetic! Without it every config file gets the contents of the _last_ config file. I have no clue why. I did a *ton* of playing with it and eventually tried out this and it works. I have no idea how this is better than the old way but it works.

I wouldn't have made the change if it were simply cosmetic.